css/highcharts.css

```diff
@@ -78,6 +78,15 @@
     /* Indicators */
     --highcharts-positive-color: #06b535;
     --highcharts-negative-color: #f21313;
+
+    /* Transparent colors for annotations */
+    --highcharts-transparent-color-0: rgba(130, 170, 255, 0.4);
+    --highcharts-transparent-color-1: rgba(139, 191, 216, 0.4);
+    --highcharts-transparent-color-2: rgba(150, 216, 192, 0.4);
+    --highcharts-transparent-color-3: rgba(156, 229, 161, 0.4);
+    --highcharts-transparent-color-4: rgba(162, 241, 130, 0.4);
+    --highcharts-transparent-color-5: rgba(169, 255, 101, 0.4);
+    --highcharts-transparent-color-6: rgba(0, 0, 0, 0.75);
```

Comment:
I'm thinking that `highcharts-transparent-color` may be a bit _too_ generic, these are not just transparent colors, but they are specifically for the annotations. They are also a gradient, at least the series 0-5 as far as I can see.

Suggestions are `highcharts-annotation-color` 0-5, then maybe replace `--highcharts-transparent-color-6` with `--highcharts-neutral-color-100` + an opacity?